pesviewer/pesviewer.py

```diff
@@ -910,9 +910,50 @@ def get_smis(m, smis, files):
                     print('Could not generate smiles for {n}'.format(n=m.name))
                 # end try
             # end for
-            return smis
+        return(smis)
     # end def
 
+    def reaction_smi():
+        """
```

Comment:
I am thinking whether this is the best place to print this file, this is in the generate_2d_depiction method, which has not much to do with the new feature. I can't see another method that would fit better.